config/clusters/2i2c-aws-us/ncar-cisl.values.yaml

```diff
@@ -0,0 +1,68 @@
+basehub:
+  userServiceAccount:
+    annotations:
+      eks.amazonaws.com/role-arn: arn:aws:iam::790657130469:role/2i2c-aws-us-ncar-cisl
+  nfs:
+    pv:
+      # from https://docs.aws.amazon.com/efs/latest/ug/mounting-fs-nfs-mount-settings.html
+      mountOptions:
+        - rsize=1048576
+        - wsize=1048576
+        - timeo=600
+        - soft # We pick soft over hard, so NFS lockups don't lead to hung processes
+        - retrans=2
+        - noresvport
+      serverIP: fs-0b70db2b65209a77d.efs.us-west-2.amazonaws.com
+      baseShareName: /
+  jupyterhub:
+    ingress:
+      hosts: [ncar-cisl.2i2c.cloud]
+      tls:
+        - hosts: [ncar-cisl.2i2c.cloud]
+          secretName: https-auto-tls
+    proxy:
```

Comment:
This is actually not required, as we do not use `proxy` anymore for HTTPS. The `proxy` stanza can be removed

Reply:
Hmmmm, I recall it was required for some reason still, but I recall thinking the same thing.

If it isn't, it should be addressed for all our hubs, because its consistently setup for all hubs after https://github.com/2i2c-org/infrastructure/pull/2349.

Reply:
Ah, I figure it may be because of this, which is something we probably can relax: https://github.com/2i2c-org/infrastructure/blob/3a6b47f68443345faff31ca805459cd52c1fdd78/helm-charts/basehub/values.schema.yaml#L258-L268

Reply:
Aaah yes we should! Not sure why that's listed as required. Am ok if you wanna do that as a separate PR, or open an issue and deal with it later as well. Not required to block this PR